### PR TITLE
Features/improve warning popup

### PR DESCRIPTION
**Pull request by SamirHassoun**
1) Changed the warning message for features tick boxes
2) improved warning popup styling
3) updated unit tests

### CODE REVIEW

File: src/Portal/Portal.UnitTests/AssessTests.cs [hunk 1]
```diff
@@ -586,7 +586,7 @@ public async Task Test_OnPostDoneAsync_given_action_done_and_features_unsubmitte
 
             Assert.AreEqual((int)VerifyCustomHttpStatusCode.WarningsDetected, response.StatusCode);
             Assert.GreaterOrEqual(_assessModel.ValidationErrorMessages.Count, 1);
-            Assert.Contains("Data Impact: All Usages Features must be submitted", _assessModel.ValidationErrorMessages);
+            Assert.Contains("Data Impact: Some Features Submitted are not ticked", _assessModel.ValidationErrorMessages);
```
Comment (247benji) [inline src/Portal/Portal.UnitTests/AssessTests.cs]:
```suggestion
            Assert.Contains("Data Impact: Some submitted features are not ticked", _assessModel.ValidationErrorMessages);
```

File: src/Portal/Portal.UnitTests/AssessTests.cs [hunk 2]
```diff
@@ -623,7 +623,7 @@ public async Task Test_OnPostDoneAsync_given_action_done_and_unsubmitted_feature
 
             Assert.AreNotEqual((int)VerifyCustomHttpStatusCode.WarningsDetected, response.StatusCode);
             Assert.GreaterOrEqual(_assessModel.ValidationErrorMessages.Count, 1);
-            Assert.False(_assessModel.ValidationErrorMessages.Contains("Data Impact: All Usages Features must be submitted"));
+            Assert.False(_assessModel.ValidationErrorMessages.Contains("Data Impact: Some Features Submitted are not ticked"));
```
Comment (247benji) [inline src/Portal/Portal.UnitTests/AssessTests.cs]:
```suggestion
            Assert.False(_assessModel.ValidationErrorMessages.Contains("Data Impact: Some submitted features are not ticked"));
```

File: src/Portal/Portal.UnitTests/VerifyTests.cs [hunk 3]
```diff
@@ -362,7 +362,7 @@ public async Task Test_OnPostDoneAsync_given_action_done_and_features_unverified
 
             Assert.AreEqual((int)VerifyCustomHttpStatusCode.WarningsDetected, response.StatusCode);
             Assert.GreaterOrEqual(_verifyModel.ValidationErrorMessages.Count, 1);
-            Assert.Contains("Data Impact: All Usages Features must be verified", _verifyModel.ValidationErrorMessages);
+            Assert.Contains("Data Impact: Some Features Verified are not ticked", _verifyModel.ValidationErrorMessages);
```
Comment (247benji) [inline src/Portal/Portal.UnitTests/VerifyTests.cs]:
```suggestion
            Assert.Contains("Data Impact: Some verified features are not ticked", _verifyModel.ValidationErrorMessages);
```

File: src/Portal/Portal.UnitTests/VerifyTests.cs [hunk 4]
```diff
@@ -397,7 +397,7 @@ public async Task Test_OnPostDoneAsync_given_action_done_and_unverified_features
 
             Assert.AreNotEqual((int)VerifyCustomHttpStatusCode.WarningsDetected, response.StatusCode);
             Assert.GreaterOrEqual(_verifyModel.ValidationErrorMessages.Count, 1);
-            Assert.False(_verifyModel.ValidationErrorMessages.Contains("Data Impact: All Usages Features must be verified"));
+            Assert.False(_verifyModel.ValidationErrorMessages.Contains("Data Impact: Some Features Verified are not ticked"));
```
Comment (247benji) [inline src/Portal/Portal.UnitTests/VerifyTests.cs]:
```suggestion
            Assert.False(_verifyModel.ValidationErrorMessages.Contains("Data Impact: Some verified features are not ticked"));
```

File: src/Portal/Portal/Helpers/PageValidationHelper.cs [hunk 5]
```diff
@@ -613,7 +613,7 @@ private async Task<bool> ValidateRecordProductAction(List<ProductAction> recordP
                     if (!dataImpacts.All(di => di.FeaturesSubmitted))
                     {
                         validationWarningMessages.Add(
-                            "Data Impact: All Usages Features must be submitted");
+                            "Data Impact: Some Features Submitted are not ticked");
```
Comment (247benji) [inline src/Portal/Portal/Helpers/PageValidationHelper.cs]:
```suggestion
                            "Data Impact: Some submitted features are not ticked");
```

File: src/Portal/Portal/Helpers/PageValidationHelper.cs [hunk 6]
```diff
@@ -622,7 +622,7 @@ private async Task<bool> ValidateRecordProductAction(List<ProductAction> recordP
                     if (!dataImpacts.All(di => di.FeaturesVerified))
                     {
                         validationWarningMessages.Add(
-                            "Data Impact: All Usages Features must be verified");
+                            "Data Impact: Some Features Verified are not ticked");
```
Comment (247benji) [inline src/Portal/Portal/Helpers/PageValidationHelper.cs]:
```suggestion
                            "Data Impact: Some verified features are not ticked");
```

File: src/Portal/Portal/Pages/DbAssessment/Verify.cshtml [hunk 7]
```diff
@@ -204,20 +204,31 @@
...
+                            <div class="col-1 mr-4">
+                                <i class="fa fa-info-circle" style="font-size: 1.50rem;"></i>
+                            </div>
+                            <div id="verifyDoneWarningMessages" class="col-11 pl-0">
+                            </div>
+                        </div>
                     </div>
-                    <div id="verifyDoneWarningMessages" class="col-11">
+                    <div class="row mt-3">
+                        <h6 class="m-0">Continue processing task?</h6>
```
Comment (247benji) [inline src/Portal/Portal/Pages/DbAssessment/Verify.cshtml]:
```suggestion
                        <h6 class="m-0">Continue signing off task?</h6>
```

File: src/Portal/Portal.UnitTests/AssessTests.cs (hunk 1 above)
Comment (SamirHassoun) [reply src/Portal/Portal.UnitTests/AssessTests.cs]: Thanks Ben. Don't agree with that, "Feature Submitted" and "Feature Verified" refer to the tick box labels.

File: src/Portal/Portal.UnitTests/AssessTests.cs (hunk 2 above)
Comment (SamirHassoun) [reply src/Portal/Portal.UnitTests/AssessTests.cs]: Thanks Ben. Don't agree with that, "Feature Submitted" and "Feature Verified" refer to the tick box labels.

File: src/Portal/Portal.UnitTests/VerifyTests.cs (hunk 4 above)
Comment (SamirHassoun) [reply src/Portal/Portal.UnitTests/VerifyTests.cs]: Thanks Ben. Don't agree with that, "Feature Submitted" and "Feature Verified" refer to the tick box labels.

File: src/Portal/Portal/Helpers/PageValidationHelper.cs (hunk 6 above)
Comment (SamirHassoun) [reply src/Portal/Portal/Helpers/PageValidationHelper.cs]: Thanks Ben. Don't agree with that, "Feature Submitted" and "Feature Verified" refer to the tick box labels.

File: src/Portal/Portal/Pages/DbAssessment/Verify.cshtml (hunk 7 above)
Comment (SamirHassoun) [reply src/Portal/Portal/Pages/DbAssessment/Verify.cshtml]: fixed, thanks

File: src/Portal/Portal.UnitTests/VerifyTests.cs (hunk 3 above)
Comment (SamirHassoun) [reply src/Portal/Portal.UnitTests/VerifyTests.cs]: Thanks Ben. Don't agree with that, "Feature Submitted" and "Feature Verified" refer to the tick box labels.

File: src/Portal/Portal/Helpers/PageValidationHelper.cs (hunk 5 above)
Comment (SamirHassoun) [reply src/Portal/Portal/Helpers/PageValidationHelper.cs]: Thanks Ben. Don't agree with that, "Feature Submitted" and "Feature Verified" refer to the tick box labels.

**Pull request merged**